.circleci/create_circleci_config.py

```diff
@@ -285,7 +285,8 @@ def job_name(self):
         "pip install --upgrade --upgrade-strategy eager pip",
         "pip install -U --upgrade-strategy eager .[sklearn,tf-cpu,torch,testing,sentencepiece,torch-speech,vision]",
         "pip install -U --upgrade-strategy eager tensorflow_probability",
-        "pip install -U --upgrade-strategy eager -e git+https://github.com/huggingface/accelerate@main#egg=accelerate",
+        # Without this we can't pin torch versions in the future
```

Comment:
maybe replace `this` with `--no-deps` and explain what `we can't pin torch versions in the future` means or replace it with a more detailed sentence?